content/page/developers/_index.md

```diff
@@ -10,25 +10,37 @@ lastmod: "2023-02-05"
 
 ## Local installation
 
-We use [poetry](https://python-poetry.org/) to manage
+1. We use [poetry](https://python-poetry.org/) to manage
 dependencies. Follow the [installation
 instructions](https://python-poetry.org/docs/#installation) to install
 poetry on your machine.
 
-Clone the repo into a local directory and `cd` into the directory.
+2. Clone the repo into a local directory and `cd` into the directory.
 
-Then run
-```
-poetry shell
-poetry install
-```
-to enter a poetry shell and install dependencies.
+3. Use `pyenv` to install the right version of python for the project (currently `3.10`):
+	```
+	brew install pyenv
+	pyenv install 3.10
```

Comment:
This is pretty Mac specific! I don't think anyone on Linux uses brew or pyenv. You can add a note that this is if you're on a mac.

Python 3.10 comes shipped with Ubuntu 22.04 ;)

Reply:
Great point, I didn't think about that.
Let me remove these instructions, and just state that `3.10` should be installed